lnrpc/routerrpc/router.proto

```diff
@@ -355,14 +355,22 @@ message PairHistory {
 }
 
 message PairData {
-    /// Time stamp of last result.
-    int64 timestamp = 1 [json_name = "timestamp"];
+    /// Time of last failure.
+    int64 fail_time = 1 [json_name = "fail_time"];
 
-    /// Last amount that was (attempted to be) forwarded.
-    int64 amt_sat = 2 [json_name = "min_penalize_amt_sat"];
+    /**
+    Lowest amount that failed to forward. This may be set to zero if the failure
+    is independent of amount.
+    */
+    int64 fail_amt_sat = 2 [json_name = "fail_amt_sat"];
```

Comment:
Could also extract this two fields into a new type `Result`.

Reply:
I considered that, but didn't want to add the nesting for only two fields.